docs/utils/index.rst

```diff
@@ -24,7 +24,6 @@ The exceptions are below:
 .. toctree::
    :maxdepth: 1
 
-   iers
```

Comment:
It has its own separate entry under `docs/index_user_docs.rst` already.

Reply:
This one is weird - I do feel we still need a reference here. Are you sure, though, that this line is the problem? I noticed that `astropy.utils.iers` is also listed in `ref_api.rst`, while `astropy.utils.data` and `astropy.utils.masked` are not - maybe removing it from `ref_api.rst` will solve the duplication?

If not, can we change it to a table of three references like you did for the others, i.e., something like

```
[no toctree]
- :ref:`astropy-utils-iers`
- :ref:`astropy-utils-data`
- :ref:`astropy-utils-masked`
```

(Never quite sure about sphinx...)

Reply:
Pretty sure this is it because the message is all about duplicate toctree entries: `docs/utils/iers.rst: document is referenced in multiple toctrees: ['index_user_docs', 'utils/index'], selecting: utils/index <- utils/iers`

Let me think about your suggestion. Thanks!

Reply:
I kept the toctree but re-added IERS just below it. Is that good enough? 😬 

Please re-review. Thanks!